backend/docs/data-model.md

```diff
@@ -0,0 +1,49 @@
+# Data Model
+
+PostgreSQL is used as data persistence layer.
+
+## Environment Variables 
+
+The following environment variables can be used to overwrite the default values:
+
+- `DB_USER`: The user for the database connection (default: `postgres`).
+- `DB_HOST`: The host for the database connection (default: `localhost`).
+- `DB_DATABASE`: The db name for the database connection (default: `postgres`).
+- `DB_PASSWORD`: The password for the database connection (default: `null`).
+- `DB_PORT`: The port for the database connection (default: `null`).
+
+## Tables
+
+All tables are defined in `lib/tables.js`.
+The key in the `lib/tables.js` file matches the table name.
+If the tables don't exist, they are created at server start with the script defined in `lib/tables.js`.
+See the create script for more details about the data types and keys.    
+
+### config
+
+`config` contains settings for the shop like the name, geographical position, etc. 
```

Comment:
This can be used for "needsRegistration" as well, I suppose?

Reply:
Yes, I thought we may need more properties and maybe even deeper structures for the shop, that's why I used a JSON structure. I guess also the UI may need more properties (labels, descriptions, longer texts, ...).